zerver/lib/actions.py

```diff
@@ -7563,14 +7565,16 @@ def do_revoke_user_invite(prereg_user: PreregistrationUser) -> None:
     Confirmation.objects.filter(content_type=content_type, object_id=prereg_user.id).delete()
     prereg_user.delete()
     clear_scheduled_invitation_emails(email)
-    notify_invites_changed(prereg_user)
+    notify_invites_changed(realm)
```

Comment:
E.g. here we were passing prereg_user after `.delete()`-ing it above. Not really a bug, because `prereg_user.realm` still works - but not a pattern we want to allow.

Reply:
Yeah, agreed this was a land mine well worth removing.